cli/bpmetadata/cmd.go

```diff
@@ -47,27 +51,53 @@ func generate(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("error getting working dir: %w", err)
 	}
 
+	//read existing metadata.yaml, if present
+	bpObj := &BlueprintMetadata{}
+	f, _ := os.ReadFile(metadataFileName)
```

Comment:
We can stat and skip if file not found

Reply:
We need to read contents of the file. Even if we stat, we'll have to read the file if its found. This way the code is kept to a min.

Reply:
I've modifed `unmarshalMetadata` to include the logic for stat and read. This reads better now.